rholang/examples/vault_demo/3.transfer_funds.rho

```diff
@@ -29,7 +29,7 @@ in {
               @vault!("transfer", to, amount, *key, *resultCh) |
               for (@result <- resultCh) {
 
-                stdout!(("Finished transfer of ", amount, "REV to", to, "result was:", result))
+                stdout!(("Finished transfer of ", amount, "nanoREV to", to, "result was:", result))
```

Comment:
its not really nanoREV :)
Options now are `stahl`, `mat`, `rho`, `revlette`

Reply:
Any suggestion for which one?

Reply:
I suggest to wait till voting app is ready and lets make an onchain vote